modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -118,12 +118,12 @@ GrainTracker::finalize()
   {
     for (const auto & grain_pair : _unique_grains)
     {
-      if (grain_pair.second->_status != INACTIVE)
+      if (grain_pair.second._status != INACTIVE)
       {
-        for (auto elem_id : grain_pair.second->_local_ids)
+        for (auto elem_id : grain_pair.second._local_ids)
         {
           mooseAssert(!_ebsd_reader || _unique_grain_to_ebsd_num.find(grain_pair.first) != _unique_grain_to_ebsd_num.end(), "Bad mapping in unique_grain_to_ebsd_num");
-          _elemental_data[elem_id].push_back(std::make_pair(_ebsd_reader ? _unique_grain_to_ebsd_num[grain_pair.first] : grain_pair.first, grain_pair.second->_var_idx));
+          _elemental_data[elem_id].push_back(std::make_pair(_ebsd_reader ? _unique_grain_to_ebsd_num[grain_pair.first] : grain_pair.first, grain_pair.second._var_idx));
```

Comment:
`emplace_back()` with passing in the two arguments will be more efficient.

What you have now creates two pairs, then calls the move constructor then destroys one of the pairs.

Using `emplace_back()` it will simply create one pair in the vector, at the right spot in memory.  Done.